commands/database-remove.js

```diff
@@ -34,19 +31,14 @@ module.exports = new Command('database:remove <path>')
 
       return new RSVP.Promise(function(resolve, reject) {
         var url = utils.addSubdomain(api.realtimeOrigin, options.instance) + path + '.json?';
-        var query = {auth: options.databaseAdminToken};
-
-        url += querystring.stringify(query);
-
-        request.del(url, {json: true}, function(err, res, body) {
-          if (err) {
-            return reject(new FirebaseError('Unexpected error while removing data', {exit: 2}));
-          } else if (res.statusCode >= 400) {
-            return reject(responseToError(res, body));
-          }
-
+        api.request('DELETE', url, {
```

Comment:
I decided to use the raw request for this, since right now api.request defaults to the admin backend  for origin, and I don't want to change the default behavior and accidentally break something else.